src/app/components/breadcrumbs/service/terra-breadcrumbs.service.ts

```diff
@@ -101,11 +103,31 @@ export class TerraBreadcrumbsService
         // breadcrumb not found
         if(isNullOrUndefined(breadcrumb))
         {
-            let label:string = this.getBreadcrumbLabel(route);
+            let activatedSnapshot:ActivatedRouteSnapshot = this.findActivatedRouteSnapshot(this.router.routerState.snapshot.root);
+            let label:string = this.getBreadcrumbLabel(route, activatedSnapshot);
             let currentContainerIndex:number = this._containers.indexOf(container);
             let previousContainer:TerraBreadcrumbContainer = this._containers[currentContainerIndex - 1];
             let parentBreadcrumb:TerraBreadcrumb = isNullOrUndefined(previousContainer) ? undefined : previousContainer.currentSelectedBreadcrumb;
-            breadcrumb = new TerraBreadcrumb(label, parentBreadcrumb, url);
+
+            let snapshotCompleteUrl:string = '';
+
+            activatedSnapshot.pathFromRoot.forEach((path:ActivatedRouteSnapshot) =>
+            {
+                if(path.url.length > 0)
+                {
+                    snapshotCompleteUrl += '/' + path.url.join('/');
+                }
+            });
+
+            if(snapshotCompleteUrl === url)
```

Comment:
Was genau hat die `snapshotCompleteUrl` was nicht bereits in der `fullUrl` drin ist?